index.html

```diff
@@ -245,5 +245,51 @@ <h3>LET’S CONNECT</h3>
         </section>
       </div>
     </main>
+    <section class="bottom-form">
+      <div class="bottom-page">
+        <h2>Contact me</h2>
+        <p class="bottom-p">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <form action="https://formspree.io/f/mgeqkbez" method="post">
+          <ul>
+            <li>
+              <input
+                type="text"
+                id="name"
+                name="user_name"
+                placeholder="Yeremias"
+                required
+                maxlength="30"
+              />
+            </li>
+            <li>
+              <input
+                type="email"
+                id="mail"
+                name="user_email"
+                placeholder="natajayanj14@gmail.com"
+                required
```

Comment:
- [ ] Well done for completing this task in style. Your design looks perfect. I have a few suggestion for you. Consider changing the  placeholder name from `Yeremias` to ` Full name` as well as the email field placehlder to `Email Address`.

Reply:
Done 👍